test/powershell/Host/Logging.Tests.ps1

```diff
@@ -237,13 +240,16 @@ $PID
         $createdEvents.Count | Should -BeGreaterOrEqual 3
 
         # Verify we log that we are executing a file
-        $createdEvents[0].Message | Should -Match ($scriptBlockCreatedRegExTemplate -f ".*/$testFileName")
+        $createdEvents[0].Message | Should -BeLike "*$testFileName*"
 
         # Verify we log that we are the script to create the scriptblock
-        $createdEvents[1].Message | Should -Match ($scriptBlockCreatedRegExTemplate -f (Get-RegEx -SimpleMatch $Script.Replace([System.Environment]::NewLine,"⏎")))
+        $createdEvents[1].Message | Select-String '[scriptblock]::create(' -SimpleMatch | Should -Not -BeNullOrEmpty
```

Comment:
We need to verify that newline got replaced with ⏎

Reply:
I feel that test is not needed, as long as expected content parts are there.